scss/components/_orbit.scss

```diff
@@ -96,7 +96,8 @@ $orbit-control-zindex: 10 !default;
   padding: $orbit-caption-padding;
 
   background-color: $orbit-caption-background;
-  color: foreground($orbit-caption-background);
+  color: color-pick-contrast($orbit-caption-background);
+  background-color: $orbit-caption-background;
```

Comment:
Ack! Must've missed that when rebasing and fixing a merge conflict. I'll clean this up…